webapp/pages/profile/_id/_slug.vue

```diff
@@ -85,6 +85,10 @@
             </ds-space>
           </template>
         </ds-card>
+        <!-- User ID -->
+        <ds-text color="softer" size="small" class="hyphenate-text" style="text-align: center;">
+          <small>{{ $t('profile.userId') }}: {{ user.id }}</small>
```

Comment:
Can we move it into the user card? I think it belongs to the user details.

Reply:
I understand your thinking @roschaefer !
But for me it is ugly additional technical information and therefore I kind of hide it.

What does the other @Human-Connection/hc-team and especially @alina-beck say?

Reply:
as I posted in zoom chat
```
I guess since I didn't see any planning on this issue, I would question if there isn't another way of dealing with the user issue

Like would having it somewhere display on hover be sufficient

is there a way that we can shorten the ids, and still have them be unique, as you have pointed out? then it might look better than these super long ids that can easily break our design...
```

Reply:
I'm with @mattwr18, I think we should have a closer look at _what exactly_ we are trying to solve with displaying the `user IDs`, _where_ we need them and _how_ users should be able to interact with them (e.g. toggle them on `click` or `hover`, in the `settings`, have them visible all the time...) and _then_ make some design decisions.

There's also this other issue dealing with `user IDs` that is closely related: #2784. And then I remember that @Mogge pointed out that the `IDs` we're using right now are not really humanly readable and thus not useful for anyone, so maybe #2786 should take priority over other `user ID` issues (also because it might make the design easier).

Reply:
The problem is that people make screenshots and send those as email as some sort of evidence for guideline violations to our moderation team. The IDs are in the URL and you cannot see them on the screenshots. So, `hover` would not make a lot of sense here.

Reply:
So we're defining a feature for users who refuse to use our built-in reporting system? :grin:

Reply:
It’s more than that:
On PlanetRomeo I had several times the experience in my live that I felt this user was already somewhen contacting me under a different name. So I made a screenshot which included their IDs at that time. Later I was able to prove for me and others that I was right.

The hover does not work for several screen shot tools I was told by @DennisHack and others …

To shorten the ID(s) there are already old issues I made long ago:

- Slugs - Permanent, unbreakable, readable and text changeable slugs for users, posts etc. #209
- 🚀 [Feature] Show User ID On User Profile Page #252

Which I have extracted into:

- 🚀 [Feature] Nano IDs For Human Readability
#2786

Reply:
I would love to have a general discussion about it @Human-Connection/hc-dev-team !